baselines/pyproject.toml

```diff
@@ -37,6 +37,10 @@ classifiers = [
     "Typing :: Typed",
 ]
 
+packages = [
+    { include = "flwr_baselines", from = "flwr_baselines" }
+]
+
```

Comment:
```suggestion
```

Just checked this. As opposed to my initial point we don't need this. The name field of the toml matches with the directory and that seems to be enough.